motion_tracker/utils/image_generator.py

```diff
@@ -176,7 +176,15 @@ def alov_generator(crops_per_image=10, batch_size=50,
 
 if __name__ == '__main__':
     my_gen = master_generator(crops_per_image=2, batch_size=50)
+    idx = 0
+    while True: 
+        X, y = next(my_gen)
+        idx += 1
+        if not idx % 100:  
+            print('iteration: {}'.format(idx))
+    '''
```

Comment:
I'll take this back out. This was just to iterate over until I found the error so I could actually debug it all.